geometry_msgs/Pose target
bool stopMotion
---
bool success
